packages/fxa-auth-server/fxa-oauth-server/lib/routes/key_data.js

```diff
@@ -27,7 +27,7 @@ module.exports = {
     payload: {
       client_id: validators.clientId,
       assertion: validators.assertion.required(),
-      scope: validators.scope,
+      scope: validators.scope.required(),
```

Comment:
This is already marked `required` in the corresponding auth-server route so I figure it's safe to make it required here as well.